app/views/wishlists/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @wishlist.name %></h1>
+
+<p>
+  <%= @wishlist.description %>
+</p>
+
+<p>
+  <strong>Your unique link for this whislist:</strong><br>
+   http://mywizlist.herokuapp.com/i/<%= @wishlist.token %>/
```

Comment:
I'm not sure, but you probably can define a custom route for generating this link.